app/pages/preview/controller.js

```diff
@@ -0,0 +1,17 @@
+import { getData } from 'buying-catalogue-library';
+import { getEndpoint } from '../../endpoints';
+import { logger } from '../../logger';
+import { getContext } from './contextCreator';
+
+export const getOrder = async ({ orderId, accessToken }) => {
+  const getOrderEndpoint = getEndpoint({ api: 'ordapi', endpointLocator: 'getOrder', options: { orderId } });
+  const orderData = await getData({
+    endpoint: getOrderEndpoint, accessToken, logger,
+  });
+
```

Comment:
Should this have a logger in saying it's come back from ordapi

Reply:
yes it should. I'll add a log message